docs/java/features/odata/use-vdm.md

```diff
@@ -64,131 +139,154 @@ service.getBusinessPartnerByKey("id")
 
 ```JAVA
 service.getBusinessPartnerByKey("id")
-    .select(BusinessPartner.FIRST_NAME)
-    .filter(BusinessPartner.FIRST_NAME.substring("substr"))
+    .select(BusinessPartner.FIRST_NAME, BusinessPartner.LAST_NAME, BusinessPartner.TO_BUSINESS_PARTNER_ADDRESS)
+    .execute(destination);
+```
+
+The above translates to the following query parameters:
+
+```TXT
```

Comment:
Here there's not good highlight option so I'd remove TXT or would downcase so that we don't spread uppercase version which doesn't work